operations/template/alert.tf

```
@@ -120,3 +120,55 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "database_token_expired_a
     ]
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "ti-log-errors-alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-log-errors-alert"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+
+  action {
+    action_group  = [azurerm_monitor_action_group.notify_slack_email[count.index].id]
+    email_subject = "TI log errors detected!"
+  }
+
+  data_source_id = azurerm_linux_web_app.api.id
+  description    = "Alert when total errors cross threshold"
+  enabled        = true
+
+  query = <<-QUERY
+      AppServiceConsoleLogs
+      | where Level == "Error"
+      and TimeGenerated >= ago(30m)
+      and TimeGenerated <= now()
+      | summarize count()
```

I think the `Level` is not our level in this case.  Looking at `operations/template/logs.tf` and the `azurerm_log_analytics_query_pack_query` resources, we need to dig into the `ResultDescription`.  Taking inspiration from that, I think the query should be something like...

```suggestion
      AppServiceConsoleLogs
      | project JsonResult = parse_json(ResultDescription) | evaluate bag_unpack(JsonResult)
      | where level == 'ERROR'
      and @timestamp >= ago(30m)
      and @timestamp <= now()
      | summarize count()
```

Also, do we need the timestamp checks given that right below the query there's a `time_window`?

---

Great question, what do you think @saquino0827 ?

---

Imo, we need both. 

The timestamp is the range of the query and the time_window is how frequently it is ran. If anything, we could refactor out the 30 so there are no gaps between the range and frequency.